### ISSUE TITLE
Video mode crop factor

### ISSUE THREAD

**Issue by peci1** [lensfun/lensfun#2384]
Hi I'm wondering if it's possible to express a different crop factor in video mode than in still image mode (e.g. my Lumix GX80 reports the exact same make/model in both and it doesn't set UniqueCameraModel).

I know there is the `<variant>` tag, but it only talks about aspect ratio. However, it's possible to take both stills and movies in e.g. 16:9, so aspect ratio itself isn't enough to distinguish it. And the video has higher crop factor because of the sensor area reserved for stabilization.

I think it would be possible to distinguish them without any code changes by just saying e.g. "append `(video)` to the variant". Is there already some standard? Or do you consider it a good solution?

Also, does anybody know if there are cameras that have the exact same crop factor in still and video mode?

**Comment by sarunasb** [comment]
@peci1 For different crop factor correction data would be different, no?

**Comment by peci1** [comment]
Yes, I assume the correction data would differ for each crop factor. That's why I'm asking how to distinguish between them.

**Comment by sarunasb** [comment]
A "video" string can probably be added for clarity, but the `<cropfactor>` element would have to specify a correct one, which would be different from that for the whole sensor.

**Comment by peci1** [comment]
Yes, of course. Many cameras even offer several aspect ratios for stills, each yielding a unique crop factor.
